internal/fields/validate.go

```diff
@@ -295,6 +328,8 @@ func createValidatorForDirectoryAndPackageRoot(fieldsParentDir string, finder pa
 	}
 
 	v.Schema = append(fields, v.Schema...)
+
+	v.LocalSchema = fields
```

Comment:
Added here for completeness, but it is not used in this validator.

Should we remove it from here?

Reply:
If not used I would say to remove it.